pandas/io/pickle.py

```diff
@@ -72,6 +101,40 @@ def read_pickle(path, compression='infer'):
     Returns
     -------
     unpickled : type of object stored in file
+
+    See Also
+    --------
+    DataFrame.to_pickle : Pickle (serialize) DataFrame object to input file
+        path.
+    Series.to_pickle : Pickle (serialize) Series object to input file path.
+    read_hdf : read from the store, close it if we opened it.
+    read_sql : Read SQL query or database table into a DataFrame.
+    read_parquet : Load a parquet object from the file path, returning a
```

Comment:
Similar comment here, "from the file path" is rather specific, would leave out that part

Reply:
amended 👍